CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
-## 0.7.6-dev0
+## 0.7.6-dev1
 
 ### Enhancements
 
+* Adds support for page numbers in `.docx` and `.doc` numbers when user or renderer
+  created page braks are present.
+
 ### Features
 
 * Updates `partition_docx` to include headers and footers in the output.
 
 ### Fixes
 
+* Page number defaults to `0` instead of `1` when page number is not present in the metadata.
+  A page number of `0` indicates that page numbers are not being tracked for the document.
```

Comment:
I wonder if `None` would be better than 0, but not a big deal either way.

Reply:
Yeah I agree on `None`, I'll make that update

Reply:
Alright got that in there. Turned out to be a little harder than expected because `_all_element_metadata` serves multiple purposes (it's the actual page incrementor when you use `partition_pdf` with `strategy="hi_res"`). May be worth a refactor at some point but seems to be working for now.